src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -310,7 +311,7 @@ public string GenerateFile()
 #if DEBUG
 					Console.WriteLine("Skipping unmodified file {0}".InvariantCultureFormat(_fileDefinition.FilePath));
 #endif
-					return File.ReadAllText(outputFile);
+					return SourceText.From(File.ReadAllText(outputFile), Encoding.UTF8);
```

Comment:
I didn't care about this code path for now. We will probably remove if/when we are using Roslyn-based generation only.

Reply:
Agreed.